decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -258,16 +320,35 @@ def gem_modifier
         @gem_modifier ||= if options[:path]
                             %(path: "#{options[:path]}")
                           elsif branch.present?
-                            %(git: "https://github.com/decidim/decidim.git", branch: "#{branch}")
+                            %(git: "#{repository}", branch: "#{branch}")
                           else
-                            %("#{Decidim::Generators.version}")
+                            %(git: "#{repository}", branch: "#{branch}" ) # TODO: remove before merge
+                            # %("#{Decidim::Generators.version}")
```

Comment:
```suggestion
                            %("#{Decidim::Generators.version}")
```
TODO: Before merging